### DISCUSSION TITLE
Which directory does code-server serve?

### DISCUSSION THREAD

**Discussion by babaric-dev** [coder/code-server#5333] [Q&A]
For any http server, there is a root directory (/). Wondering what is the equivalent in code-server?

PS I need to some files to the / because of an authentication I need.

**Comment by jsjoeio** [comment]
🤔 I think it depends on the setup. How are you using code-server?

**Reply by babaric-dev** [reply]
Sorry for the very late reply. I am installing code server in `~/code-server/`. Im on Linux

**Reply by jsjoeio** [reply]
No worries. I am not sure what the default directory is. Maybe `HOME`? What happens when you run `code-server` and then type `pwd` in the Integrated Terminal? You should be able to pass a directory to `code-server /some/path` and it will start there.

**Reply by lg-nineoneninethreenine** [reply]
I got $HOME/.local/bin/code-server, could help here. Bet it's the same all around since code-server doesn't actually go to /use/bin since it doesn't run in sudo mode. So after that, we have to add .local/bin to path and run code-server.

**Reply by babaric-dev** [reply]
I got `$HOME/code-files` for the `pwd` command in the integrated terminal. I run code-server with `code-server --user-data-dir $HOME/code-data $HOME/code-files` and `realpath $(which code-server)` returns `$HOME/code/lib/bin/code-server`.

**Reply by lg-nineoneninethreenine** [reply]
Wait, so you downloaded a tarball or installed it via the one-line command? It's weird seeing it there to me.

**Reply by babaric-dev** [reply]
I downloaded the tarball from the GitHub releases
